### DISCUSSION TITLE
I want to do a local hosting.

### DISCUSSION THREAD

**Discussion by Crisespecial** [mm201/pkmnFoundations#119] [General]
Basically im looking if i can host the private server as i wanna have admin rights and i was gonna use an offline router so i wanted to have my own private offline server like you can do with other stuff such as SanicBall and others but i want to have a server that can be used for private reason instead of the public server. How do i do that?

**Comment by mm201** [comment]
It isn't entirely trivial but guides do exist. You need to run a local altWFC instance along with PCN. There's a [somewhat old but probably still valid guide](https://github.com/mm201/pkmn-classic-framework/wiki/Creating-a-development-server) in the wiki.
